itest/itest.sh

```diff
@@ -82,18 +82,23 @@ function main {
         -v $TOPLEVEL/itest/example_config.yml:/work/config.yml \
         -v $TOPLEVEL/$FIFO_NAME:/work/outfile \
         pidtree-itest -c /work/config.yml -f /work/outfile
-  elif [[ "$@" = "--deb" ]]; then
-    mkdir -p itest/dist
-    rm -f itest/dist/*.deb
-    cp $(ls -t packaging/dist/*.deb | head -n 1) itest/dist/  
-    docker build -t pidtree-itest-deb -f itest/Dockerfile.ubuntu_xenial itest
+  elif [[ "$1" = "ubuntu_xenial" || "ubuntu_bionic" ]]; then
```

Comment:
There is a `"$1" = ` missing here, otherwise this is always true.

Reply:
Well spotted